vsts/windows_csharp.cmd

```diff
@@ -9,20 +9,33 @@ for %%i in ("%build-root%") do set build-root=%%~fi
 
 cd %build-root%
 
-rem Build Debug + run UTs
-call build.cmd -clean -configuration Debug
-if errorlevel 1 goto :eof
+rem Handle Windows Desktop vs Windows IoT Core
+for /f "tokens=3" %%a in ('reg query "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion"  /V EditionID  ^|findstr /ri "REG_SZ"') do set EditionID=%%a
```

Comment:
Explainiation required - what is handled here?

Reply:
The batch script reads the reg key HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\EditionID into a variable _EditionID_ and depending if its value is "IoTUAP" (aka Windows IoT Core) or not (Windows Desktop), the execution continues differently with the conditional _block if/else_.
The _else block_ that handles Windows Desktop is unchanged. The newly added _if block_ in case of Windows IoT Core is just a little different: added _-nobuild -nopackage_ parameters and the use of _Simulator.exe_ is not needed (Windows IoT Core supports TPM out of the box)